caikit/runtime/train.py

```diff
@@ -295,6 +314,13 @@ def main() -> int:
             args.model_name,
         ):
             future = train(module, wait=True, **train_kwargs)
+
+            # The .complete file indicates all files have completed
```

Comment:
I think this should probably be done after the termination log too. You could probably write a special `exit_complete` function that replaces all the places you call `exit` and does this `.complete` file then exits. Since it's supposed to be "all done touching disk" and since the termination log may be an important artifact for the job's parent to parse, I think it should be the very last thing that mutates the disk.